docs/preview/features/secret-store/provider/configuration.md

```diff
@@ -32,7 +32,11 @@ public class Program
                    .ConfigureSecretStore((HostBuilderContext context, IConfiguration config, SecretStoreBuilder builder) =>
                    {
 #if DEBUG
+                       // Uses the built `IConfiguration` as a secret provider.
                        builder.AddConfiguration(config);
+
+                       // Uses the built `IConfiguration` as secret provider, using `:` instead of `.` when looking up secrets.
+                       builder.AddConfiguration(config, secretName => secretName.Replace(".", ":"));
```

Comment:
Good example! ❤️ Maybe add an example?
```suggestion
                       // Example - When looking up Queue.Name it will be changed to queue:name
                       builder.AddConfiguration(config, secretName => secretName.Replace(".", ":").ToLower());
```